package.json

```diff
@@ -23,8 +23,8 @@
     "@fortawesome/fontawesome-svg-core": "^6.2.1",
     "@fortawesome/free-solid-svg-icons": "^6.2.1",
     "@fortawesome/react-fontawesome": "^0.2.0",
-    "@mantine/core": "^4.2.12",
-    "@mantine/hooks": "^4.2.12",
+    "@mantine/core": "^5.10.5",
+    "@mantine/hooks": "^5.10.5",
```

Comment:
Are we sure we want to use this version of mantine? Usually in libraries the lowest compatible version is better. If v4 works for us and does not have any major issues in performance, we can use v4

Reply:
4 did cause problems actually, and some of the components we use (mostly the Popover) were updated in 5. But it definitely doesn't need to be exactly 5.10.5, that was just the most recent